default.json

```diff
@@ -28,7 +28,8 @@
   "extends": [
     "config:base",
     "default:automergeDigest",
-    "docker:enableMajor"
+    "docker:enableMajor",
+    ":semanticCommitsDisabled"
```

Comment:
Both `automergeDigest` and `semanticCommitsDisabled` are default presets.

Both ways to reference them are correct:

> Special note: the :xyz naming convention (with : prefix) is a special shorthand for the default presets. i.e. :xyz is equivalent to default:xyz.

-- [Shareable Config Presets | Renovate Docs](https://docs.renovatebot.com/config-presets/#example-configs)

But I think we should use only one way in our preset to avoid confusion.

Reply:
Right, I failed to understand why this preset's name was just starting with a colon. I will update the PR.

Reply:
In case my opinion is relevant, I highly appreciate @cfra used the more explicit syntax, as I find it less confusing. 
IMHO, syntactic sugar has its rightful places, but config files or alike are not one of them.

Reply:
@cybniv Sorry, I don't get your comment. 😕

Reply:
> Both `automergeDigest` and `semanticCommitsDisabled` are default presets.
> 
> Both ways to reference them are correct:
> 
> > Special note: the :xyz naming convention (with : prefix) is a special shorthand for the default presets. i.e. :xyz is equivalent to default:xyz.
> 
> -- [Shareable Config Presets | Renovate Docs](https://docs.renovatebot.com/config-presets/#example-configs)
> 
> But I think we should use only one way in our preset to avoid confusion.

@keimlink in your previous comment (above), you pointed out that there's more than one "correct" way to do it, meaning one can use the shorthand way for default presets, i.e. `:xyz:` which is equivalent to the more explicit way `default:xyz`. I just said that I think the latter is better and why.

Reply:
@cybniv I should have been more precise about what caused my confusion.

You wrote:

>  I highly appreciate @cfra used the more explicit syntax

But the diff we're discussing shows exactly the opposite…

Reply:
@keimlink Now I'm completely confused. Of course @cfra used the more explicit syntax `default:xyz`

Reply:
@cybniv Please have a look at the diff above [my first comment](https://github.com/PicturePipe/renovate-config/pull/243#discussion_r405071671) which started the discussion.

Reply:
@keimlink Yes, exactly, at that point you have rightly criticised one should use decide for one syntax. @cfra did then decide for the more explicit syntax. I then said I appreciate that.